### DISCUSSION TITLE
Clickable ID on FlowViz

### DISCUSSION THREAD

**Discussion by mariaorlando** [nbrown02/FlowViz#125] [Ideas]
Hi Team,

It would be great to have a clickable ID for all work items showing up on the Teams' FlowViz. 

It will facilitate PMs, developers, SMs and Agile Coaches when checking metrics and see more details about the work item instead of searching for each of them on the Azure Board. This is beneficial especially for those teams with a big backlog. 

Do you think it is a milestone we can achieve?

Thank you!

Best,
Maria

**Comment by nbrown02** [comment]
Hey Maria,

Thanks for reaching out. Many of the charts on Dashboard and WIP pages have drillthrough, which is the only way you could really get to the item in ADO. Right click on a data-point then drill-through, then the table view of the drill-through page is a hyperlink to the item in ADO...

![Drill](https://github.com/user-attachments/assets/6b87fa5c-fba0-44aa-a5b0-93901b716251)

**Reply by mariaorlando** [reply]
Hi @nbrown02 

Thanks for this clarification :-) This is great!

I was talking more about Work Item Age section. Is it possible to make it clickable from there, at placing the mouse cursor over the work item without drilling through?

**Reply by nbrown02** [reply]
Unfortunately not no, I want it to do that but sadly not possible in the out the box Power BI visuals...tbh I'm not even sure a custom visual could do it!

**Reply by mariaorlando** [reply]
Ohhh thanks anyway @nbrown02 ! 

It would have been an amazing improvement, because it reduces the clicks for the final user! Who knows if in the future it will be possible (I am optimistic lol)
